src/Service/ConfigurationLocator.php

```diff
@@ -82,15 +83,16 @@ public function locate(CorsMetadata $metadata): ?ConfigurationInterface
 
     private function configurationFromRoute(RouteResult $result): RouteConfigurationInterface
     {
-        $allowedMethods = $result->getAllowedMethods();
-        if ($allowedMethods === Route::HTTP_METHOD_ANY) {
```

Comment:
Can re-add on next release of psalm.
https://github.com/vimeo/psalm/issues/4061

Reply:
I think I'd rather suppress the Psalm error and have a `@todo` to update, as the "solution" hard-codes what it's looking for, versus using a constant value.